R/utilities.R

```diff
@@ -40,3 +40,32 @@ netEnumName <- function(enumType, enumValue) {
   netTypeObj <- rClr::clrGetType(enumType)
   rClr::clrCallStatic("System.Enum", methodName = "GetName", netTypeObj, enumValue)
 }
+
+
+#' Clears the memory used by all underlying objects
+#' @details The function aims at clearing the memory used by object references
+#' allocated during some workflows. The memory should typically be freed automatically.
+#' when the system is under memory pressure or when the garbage collection is kicking
+#' in. However it may be necessary sometimes to explicitly start the garbage collection
+#' process
+#'
+#' @param clearSimulationsCache optional - Should the simulation cache also be cleared? Default is `FALSE`
+#'
+#' @examples
+#' # This will clear the memory and also clear the simulations cache but leave
+#' # the environment intact.
+#' clearMemory(clearSimulationsCache = TRUE)
+#'
+#' @export
+clearMemory <- function(clearSimulationsCache = FALSE) {
```

Comment:
@Yuri05 Your suggestion with clearing the environment 
1 - did not work (maybe it works in the terminal but here it clears the env of the function)
2 - seems to be very dangerous

So I am removing it for now

Reply:
hmm, ok

Reply:
I think clearing the env is not what we want... Just invoking GC on R and .NET.

Reply:
My 2 cents: Not a good idea to have abbreviations in function names. 
If we _do_ want to change the name, I'd argue for something like `forceGarbageCollection()`.